RecoParticleFlow/PFClusterProducerAlpaka/plugins/AlpakaPFCommon.h

```diff
@@ -0,0 +1,91 @@
+#ifndef RecoParticleFlow_PFClusterProducerAlpaka_plugins_AlpakaPFCommon_h
+#define RecoParticleFlow_PFClusterProducerAlpaka_plugins_AlpakaPFCommon_h
+
+#include <cmath>
+
+namespace PFClustering {
+  namespace common {
+    enum PFLayer {
+      PS2 = -12,
+      PS1 = -11,
+      ECAL_ENDCAP = -2,
+      ECAL_BARREL = -1,
+      NONE = 0,
+      HCAL_BARREL1 = 1,
+      HCAL_BARREL2 = 2,
+      HCAL_ENDCAP = 3,
+      HF_EM = 11,
+      HF_HAD = 12,
+      HGCAL = 13  // HGCal, could be EM or HAD
+    };
+
+    struct TimeResConsts {
+      float corrTermLowE;
+      float threshLowE;
+      float noiseTerm;
+      float constantTermLowE2;
+      float noiseTermLowE;
+      float threshHighE;
+      float constantTerm2;
+      float resHighE2;
+    };
+
+    struct PosCalcConfig {
+      float minAllowedNormalization;
+      float logWeightDenominatorInv;
+      float minFractionInCalc;
+      TimeResConsts timeResEndcap;
+      TimeResConsts timeResBarrel;
+
+      PosCalcConfig() : minAllowedNormalization(-1.0), logWeightDenominatorInv(-1.0), minFractionInCalc(-1.0){};
+
+      PosCalcConfig(float _minAllowedNormalization,
+                    float _logWeightDenominatorInv,
+                    float _minFractionInCalc,
+                    TimeResConsts _timeResEndcap,
+                    TimeResConsts _timeResBarrel)
+          : minAllowedNormalization(_minAllowedNormalization),
+            logWeightDenominatorInv(_logWeightDenominatorInv),
+            minFractionInCalc(_minFractionInCalc),
+            timeResEndcap(_timeResEndcap),
+            timeResBarrel(_timeResBarrel){};
+    };
+
+    // Config parameters for ECAL2DPositionCalcWithDepthCorr
+    struct ECALPosDepthCalcConfig {
+      float minAllowedNormalization;
+      float T0_ES;
+      float T0_EE;
+      float T0_EB;
+      float X0;
+      float minFractionInCalc;
+      float W0;
+
+      ECALPosDepthCalcConfig()
+          : minAllowedNormalization(-1.0),
+            T0_ES(-1.0),
+            T0_EE(-1.0),
+            T0_EB(-1.0),
+            X0(-1.0),
+            minFractionInCalc(-1.0),
+            W0(-1.0){};
```

Comment:
Would using default member initializers and relying on aggregate initialization work?